python-stream/receive.py

```diff
@@ -0,0 +1,30 @@
+import asyncio
+import signal
+
+from rstream import AMQPMessage, Consumer, MessageContext
+
+STREAM_NAME = "hello-python-stream"
+
+
+async def receive():
+    consumer = Consumer(host="localhost", username="guest", password="guest")
+    await consumer.create_stream(STREAM_NAME, exists_ok=True)
```

Comment:
Can you please add the size retention https://github.com/rabbitmq/rabbitmq-tutorials/pull/383/files#diff-aaa61986b4ca5dbb6e120b1304f70f4014885275501a001306d4879b5389b017R30 ? like 2 GB ? Thank you